pipeline.yml

```diff
@@ -32,3 +32,4 @@ jobs:
         params:
           manifest: grafana-paas-git/manifest.yml
           show_app_log: true
+          docker_password: ((dockerhub-password))
```

Comment:
non-blocking: this var name isn't ideal, I can imagine a future where we want multiple dockerhub password vars in this team and this one just happens to be for the `gdsobserve` user

Reply:
Yeah, I set it just for the grafana pipeline.